CONTRIBUTORS.yaml

```
@@ -914,6 +914,10 @@ kxk302:
     email: kxk302@gmail.com
     joined: 2021-02
 
+kysrpex:
+    name: José Manuel Domínguez
+    joined: 2023-10
+
```

@kysrpex do you have any other information you'd like in your public GTN profile? E.g. an ORCID?

```suggestion
kysrpex:
    name: José Manuel Domínguez
    joined: 2023-10
    elixir_node: de

```

---

It's fine like this :+1: I know now where to add stuff if want to :)

---

fair enough!